docs/resources/alert_method_email.md

```diff
@@ -34,18 +32,18 @@ resource "nobl9_alert_method_email" "this" {
 
 ### Required
 
-- `body` (String) The Body of the email alert. For the format of the body and the list of variables that you can define, refer to the [Nobl9 documentation](https://docs.nobl9.com/Alerting/Alert_methods/email-alert#yaml-configuration).
 - `name` (String) Unique name of the resource, must conform to the naming convention from [DNS RFC1123](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names).
 - `project` (String) Name of the Nobl9 project the resource sits in, must conform to the naming convention from [DNS RFC1123](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names).
-- `subject` (String) The Subject of the email alert.
 - `to` (List of String) Recipients. The maximum number of recipients is 10.
 
 ### Optional
 
 - `bcc` (List of String) Blind carbon copy recipients. The maximum number of recipients is 10.
+- `body` (String, Deprecated) Deprecated value that was used as the body template of email alert. It's not used anywhere but kept for backward compatibility.
```

Comment:
Maybe move body next to subject at the end of the list? Or maybe even dedicated section "Deprecated" (next to Optional?)

Reply:
It's generated automatically, and sorted alphabetically for the ease of searching through the docs. I'd say let's leave it :D

Reply:
Nice ❤️